lib/ddtrace/contrib/active_job/configuration/settings.rb

```diff
@@ -25,6 +25,7 @@ class Settings < Contrib::Configuration::Settings
 
           option :service_name, default: Ext::SERVICE_NAME
           option :error_handler, default: Datadog::Tracer::DEFAULT_ON_ERROR
+          option :log_injection, default: false
```

Comment:
Could you document this new option here: https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#active-job
(No need to worry about the missing `error_handler`)

Reply:
👍🏻 done! I used the same content as the Rails `log_injection` Rails option.